src/api/client/Claims.ts

```diff
@@ -483,4 +491,54 @@ export class Claims {
    *  - a custom claim type with the same `name` already exists
    */
   public registerCustomClaimType: ProcedureMethod<RegisterCustomClaimTypeParams, BigNumber>;
+
+  /**
+   * Retrieves a custom claim type based on its name.
+   *
+   * @param name - The name of the custom claim type to retrieve.
+   *
+   * @returns A promise that resolves to the `CustomClaimTypeToHuman` object if found, or null otherwise.
+   */
+  public async getCustomClaimTypeByName(name: string): Promise<CustomClaimTypeToHuman | null> {
+    const {
+      context: {
+        polymeshApi: {
+          query: { identity },
+        },
+      },
+    } = this;
+
+    const customClaimTypeIdOpt = await identity.customClaimsInverse(name);
+
+    if (!customClaimTypeIdOpt.isSome) {
```

Comment:
```suggestion
    if (customClaimTypeIdOpt.isEmpty) {
```

Reply:
My nitty preference is `isNone` for options, but I looked it up and `isEmpty` and `isNone` are the same thing for `Option`. I agree about getting rid of the `!` is an improvement